modules/volunteers/admin/sitesupport/giftTable.js

```diff
@@ -254,34 +325,54 @@ export default {
       } else {
         this.decendingSort = false;
       }
+
       const sortDirection = this.decendingSort ? -1.0 : 1.0;
       this.sortColumn = column.value;
 
-      this.records.sort((a,b) => {
-        let result = (parseFloat(a[column.value]) - parseFloat(b[column.value])) * sortDirection;
-        if (column.value === 'remaining') {
-          result = ((a.inventory - a.claimed) - (b.inventory - b.claimed)) * sortDirection;
-        } else if (column.type === String) {
-          result = a[column.value].localeCompare(b[column.value]) * sortDirection;
+      sortPrizes(this.records, column, sortDirection);
+      this.groups.forEach((group) => {
+        sortPrizes(group.items, column, sortDirection);
+      });
+    },
+    printGroupValue(record, column) {
+      if (column.value === 'value') {
+        if (this.groups[record.reward_group.id].valueLow == this.groups[record.reward_group.id].valueHigh) {
+          return this.$parent.printHours(this.groups[record.reward_group.id].valueLow);
         }
+        return '<' + this.$parent.printHours(this.groups[record.reward_group.id].valueLow) + ' -- ' +
+          this.$parent.printHours(this.groups[record.reward_group.id].valueHigh) + '>';
+      }
 
-        if (result === 0.0) {
-          if (a.name ===  b.name) {
-            return (parseFloat(a.value) - parseFloat(b.value)) * sortDirection;
+      if (column.value === 'promo') {
+        if (this.groups[record.reward_group.id].promo !== null) {
+          if (parseInt(record.promo) === 1) {
+            return 'yes';
           }
-          return a.name.localeCompare(b.name) * sortDirection;
+          return 'no';
         }
-        return result;
-      });
+        return 'varies';
+      }
 
-      this.records.sort((a,b) => {
-        if (a.reward_group !== null && b.reward_group === null) {return -1;}
-        if (b.reward_group !== null && a.reward_group === null) {return 1;}
-        if (b.reward_group === null && a.reward_group === null) {return 0;}
-        if (parseInt(b.reward_group.id) === parseInt(a.reward_group.id)) {return 0;}
-        return parseInt(a.reward_group.id) - parseInt(b.reward_group.id);
-      });
-    }
+      if (column.value === 'inventory') {
+        const str = 'Click to ' + ((this.groups[record.reward_group.id].open) ? 'close' : 'open');
+        return str;
```

Comment:
We could potentially simplify this with template literals:

```suggestion
        return `Click to ${this.groups[record.reward_group.id].open ? 'close' : 'open'}`;
```